### PR TITLE
[RUN 3912] Add information about SBOM in Dependency Synchronization section

### PR DESCRIPTION
**Pull request by akkrishnakumar**
Add information about SBOM in Dependency Synchronization section

### CODE REVIEW

File: content/en/docs/refguide/java-programming/managed-dependencies.md [hunk 1]
```diff
@@ -57,6 +57,8 @@ You can trigger a manual synchronization in the menu **App** > **Synchronize dep
 
 Once Gradle has resolved and downloaded all dependencies successfully, it places them in the `vendorlib` directory. This directory is used to store `.jar` files of all the managed dependencies. Unlike the `userlib` folder, which needs to be updated manually, the `vendorlib` folder is completely managed by Studio Pro. Files here will automatically be added and removed based on the specified dependencies. The `vendorlib` folder will be committed to your version control just like the `userlib`.
 
+We also generate the `vendorlib-sbom.json` file. This is the Software Bill of Materials of the `vendorlib` directory.
```
Comment (MarkvanMents) [inline content/en/docs/refguide/java-programming/managed-dependencies.md]: Should we say where this file is? I presume it is in the `/vendorlib` folder?

**Review by MarkvanMents — CHANGES_REQUESTED** [review]
Hi Akhil (@akkrishnakumar)
Just one question about where this json file is. Should we say where the customer might find it or see it as part of a commit?
If that is irrelevant, then I can merge this - unless it is for a future release.

File: content/en/docs/refguide/java-programming/managed-dependencies.md (hunk 1 above)
Comment (akkrishnakumar) [reply content/en/docs/refguide/java-programming/managed-dependencies.md]: Yes, that sounds more explicit.

Comment (MarkvanMents) [reply content/en/docs/refguide/java-programming/managed-dependencies.md]: Updated here: 263b2e145df31b6d4f997f7ddaf0dc44bdf80c34

**Review by MarkvanMents — APPROVED** [review]
Approving and merging immediately as discussed on Slack.

**Pull request merged**